py/server/tests/test_table.py

```diff
@@ -237,6 +237,13 @@ def test_head_tail_pct(self):
                 result_table = op(self.test_table, pct=0.1)
                 self.assertEqual(result_table.size, self.test_table.size * 0.1)
 
+    def test_slice_pct(self):
+        ops = [Table.slice_pct]
+        for op in ops:
```

Comment:
Not a big deal but are we going to have other similar ops that we might want to group test them? If not, then it is better to remove the iteration.